src/ASM/LR/ASMu2Dmx.h

```diff
@@ -116,15 +116,16 @@ class ASMu2Dmx : public ASMu2D, private ASMmxBase
   //! \param[in] gpar Parameter values of the result sampling points
   //! \param[in] regular Flag indicating how the sampling points are defined
```

Comment:
Since removed as argument, also remove the doxy for `regular`. Consider removing it from all ASMu#Dmx methods, since used for tensor spline meshes only.